### PR TITLE
Vet clinic database: database performance audit

### PR DESCRIPTION
**Pull request by abdozayan12**
Screenshot 2023-08-21 172600

Before:
Screenshot 2023-08-21 172438

After 👍 
Screenshot 2023-08-21 173018

Before
Screenshot 2023-08-22 084920

After 
Screenshot 2023-08-22 085046

Before
Screenshot 2023-08-21 172841

 After
Screenshot 2023-08-21 173521

### CODE REVIEW

**Review by rloterh — COMMENTED** [review]
Hi Team, 

While you made a great effort in this project, unfortunately, I cannot proceed to review your code.

## Invalid Code Review Request

Attached screenshots of query results are inaccessible. Kindly ensure that the attached screenshots in the Pull Request descriptions are displaying correctly.


**Screenshot are inaccessible/Not displaying correctly:**

image

**Clicking on each attached screenshot:**

image


Your Code Review Request will be marked as invalid in your Dashboard, so please submit a new one once you are ready 🙏

Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.
Please, remember to tag me in your question so I can receive the notification.

**_Please, do not open a new Pull Request for re-reviews. You should use the same Pull Request submitted for the first review, either valid or invalid unless it is requested otherwise._**

--------

_Invalid Code Review Request **does not count** into the code reviews limit._

File: queries.sql [hunk 1]
```diff
@@ -152,4 +152,8 @@ JOIN vets ON visits.vet_id = vets.id
 WHERE vets.name = 'Maisy Smith'
 GROUP BY species.name
 ORDER BY visit_count DESC
-LIMIT 1;
\ No newline at end of file
+LIMIT 1;
+
+SELECT COUNT(*) FROM visits where animal_id = 4;
+SELECT * FROM visits where vet_id = 2;
+SELECT * FROM owners where email = 'owner_18327@mail.com';
```
Comment (Gambit142) [inline queries.sql:159]: - [ ] The screenshot that you attached for the query below depicting that you added the index to the `vet_id` column of the `visits` table does not show that the index was added through the sequential scan:

```sql
SELECT * FROM visits where vet_id = 2;
```

### The Screenshot

image

I would advise you to remove the index for this column and add it again. Please take screenshots of the `explain analyze` output one more time after removing and adding it again. 👍 😎

**Review by Gambit142 — CHANGES_REQUESTED** [review]
Hi @abdozayan12 @ichsansandy 👋, 

Good job so far!

## Highlights
- Added necessary columns ✔️ 
- Created huge data for tables ✔️ 
- Wrote the required queries ✔️ 

You still need to work on some issues to go to the next project, but you are almost there!

## Required Changes ♻️
_Kindly check the comments under the review._

## Optional suggestions
_Every comment with the [OPTIONAL] prefix is not crucial enough to stop the approval of this PR. However, I strongly recommend considering them as they can improve your code._

You can also consider the following:

N/A

Cheers, and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.

#### Please tag me in your question so I can receive the notification. You can tag me by typing @Gambit142

**_Please, do not open a new Pull Request for re-reviews. You should use the same Pull Request submitted for the first review, either valid or invalid unless it is requested otherwise._**

------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561) you have a limited number of reviews per project (check the exact number in your Dashboard). If you think the code review was unfair, request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._
```

File: queries.sql (hunk 1 above)
Comment (abdozayan12) [reply queries.sql:159]: I've removed the INDEX and recreated it
Screenshot 2023-08-21 224720

Screenshot 2023-08-21 223627
Screenshot 2023-08-21 223653

Comment (abdozayan12) [reply queries.sql:159]: @Gambit142

**Review by prolajumokeoni — APPROVED** [review]
Hi @Team

Your project is complete!  There is nothing else to say other than... it's time to merge it :shipit: 
Congratulations! 🎉 
## Highlights
- Changes requested have been implemented   ✅ 
 

## Optional suggestions
_Every comment with the [OPTIONAL] prefix is not crucial enough to stop the approval of this PR. However, I strongly recommend you to take them into account as they can make your code better._

You can also consider:
- N/A

Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.
Please, remember to tag me in your question so I can receive the notification @prolajumokeoni.

------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561)  you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

**Pull request merged**